datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py

```diff
@@ -244,94 +242,88 @@ def metadata(check, check_duplicates):
-                if not row['description']:
-                    empty_warning_count['description'] += 1
-                # exceeds max allowed length of description
-                elif len(row['description']) > MAX_DESCRIPTION_LENGTH:
-                    errors = True
-                    echo_failure(
-                        f"{current_check}:{line} `{row['metric_name']}` exceeds the max length: "
-                        "{MAX_DESCRIPTION_LENGTH} for descriptions."
-                    )
-                if row['interval'] and not row['interval'].isdigit():
-                    errors = True
-                    echo_failure(f"{current_check}: interval should be an int, found '{row['interval']}'")
+            # metric_name header
+            if metric_prefix:
+                if not row['metric_name'].startswith(metric_prefix):
+                    prefix = row['metric_name'].split('.')[0]
+                    metric_prefix_count[prefix] += 1
+            else:
+                errors = True
+                if not metric_prefix_error_shown and current_check not in PROVIDER_INTEGRATIONS:
+                    metric_prefix_error_shown = True
+                    echo_failure(f'{current_check}:{line} metric_prefix does not exist in manifest')
+
+            # metric_type header
+            if row['metric_type'] and row['metric_type'] not in VALID_METRIC_TYPE:
+                errors = True
+                echo_failure(f"{current_check}:{line} `{row['metric_type']}` is an invalid metric_type.")
+
+            # unit_name header
+            if row['unit_name'] and row['unit_name'] not in VALID_UNIT_NAMES:
+                errors = True
+                echo_failure(f"{current_check}:{line} `{row['unit_name']}` is an invalid unit_name.")
+
+            # orientation header
+            if row['orientation'] and row['orientation'] not in VALID_ORIENTATION:
+                errors = True
+                echo_failure(f"{current_check}:{line} `{row['orientation']}` is an invalid orientation.")
+
+            # empty required fields
+            for header in REQUIRED_HEADERS:
+                if not row[header]:
+                    empty_count[header] += 1
+
+            # empty description field, description is recommended
+            if not row['description']:
+                empty_warning_count['description'] += 1
+            # exceeds max allowed length of description
+            elif len(row['description']) > MAX_DESCRIPTION_LENGTH:
+                errors = True
+                echo_failure(
+                    f"{current_check}:{line} `{row['metric_name']}` exceeds the max length: "
+                    "{MAX_DESCRIPTION_LENGTH} for descriptions."
+                )
+            if row['interval'] and not row['interval'].isdigit():
+                errors = True
+                echo_failure(f"{current_check}: interval should be an int, found '{row['interval']}'")
```

Comment:
GitHub UI is overkilling the diff here — this is just an de-indent of this entire block due to switching to `read_metadata_rows()`.

Reply:
Thanks, changing config to hide whitespace only changes helps

Reply:
Didn't know about that feature! :+1: